### PR TITLE
docs(open source): add required files for open source

### LINKED ISSUE #114: Add required files
This repo lacks maintainers file, contributor guidelines, and a license file. Please add them -- thanks!

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #114

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Damián García <damian.garcia@zalando.de>
```
Comment (frontendherodk) [inline MAINTAINERS]: Im thinking if we should always just include the whole team in all our repos?

**Comment by frontendherodk** [comment]
👍

File: MAINTAINERS (hunk 1 above)
Comment (dami-gg) [inline MAINTAINERS]: I just copied and pasted it from package.json. It's a Norris team component so nobody will ever read this file, they'll directly talk to the team, but has to be there to be compliant you know... ¬¬

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**